### DISCUSSION TITLE
Xyna: Workflow-Diff erstellen

### DISCUSSION THREAD

**Discussion by katharina-walter** [Xyna-Factory/xyna-factory#2319] [Ideas]
Im Gespräch mit Team 1, 2 & 3 hat sich ergeben, dass die hilfreichste Xyna-Erweiterung bzgl. kollaborativem Arbeiten eine Anzeige von Unterschieden zwischen zwei Workflow-Versionen wäre.

Anforderungen

- zwei Workflows nebeneinander anzeigen
  - eigener Stand
  - remote Stand
- Unterschiede müssen hervorgehoben werden, unterscheidbar ob eigene oder remote Änderungen
- eigener Stand muss änderbar sein

**Comment by PatrickVonMassow-GIP** [comment]
This is an extensively change that needs to be discussed before creating tickets for the implementation -> converting the issue into a discussion.
